src/ol/interaction/translate.js

```diff
@@ -234,6 +240,35 @@ ol.interaction.Translate.prototype.setHitTolerance = function(hitTolerance) {
 
 
 /**
+ * @inheritDoc
+ */
+ol.interaction.Translate.prototype.setMap = function(map) {
+  var oldMap = this.getMap();
+  ol.interaction.Pointer.prototype.setMap.call(this, map);
+  this.updateState_(oldMap);
+};
+
+
+/**
+ * @param {ol.Map} oldMap Old map.
+ * @private
+ */
+ol.interaction.Translate.prototype.updateState_ = function(oldMap) {
```

Comment:
@tschaub : When the `active` property is changed to `false`, `getMap()` should returns something, so `oldMap` would not be used. But if nonetheless `getMap()` returns `null`, I don't see any cases where `previousCursor_` would be defined.